src/Orc.Controls/Controls/StepBar/ViewModels/StepBarItemViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Orc.Controls
+{
+    using Catel.MVVM;
+
+    public class StepBarItemViewModel : ViewModelBase
```

Comment:
Add a ctor with IStepBarItem as model. We might want to add more functionality here in the future.